problem.py

```diff
@@ -1,20 +1,20 @@
 import json
 import os
+import random
 import typing
 from dataclasses import dataclass
-from itertools import tee
 from os.path import join as pjoin
 
 import numpy as np
 import pandas as pd
-
 from rampwf.prediction_types.base import BasePrediction
 from rampwf.score_types import BaseScoreType
 from rampwf.workflows import Estimator
 from sklearn.model_selection import GroupShuffleSplit
 
 DATA_HOME = "data"
 RANDOM_STATE = 777
+random.seed(RANDOM_STATE)
```

Comment:
can we rather use np.random.RandomState to avoid affected the global state of random?

Reply:
Thanks. Solved in #15